pkgs/misc/vscode-extensions/default.nix

```diff
@@ -839,9 +839,8 @@ let
         mktplcRef = {
           name = "metals";
           publisher = "scalameta";
-          version = "1.10.3";
-          sha256 = "0m4qm1z1j6gfqjjnxl8v48ga7zkaspjy3gcnkrch3aj4fyafjl09";
-        };
```

Comment:
I think you accidentally delete this bracket.

Reply:
Fixed!